utils/devServerProxy/proxy.js

```diff
@@ -41,6 +41,9 @@ const devAccountMap = {
     DEVXBDEPLQEQZ: ['DE', ['gpl'], 'gplq_eqz-non-de'],
     DEVXBDEPLQGTZ: ['DE', ['gpl'], 'gplq_gtz-non-de'],
 
+    DEV000000PI30: ['DE', ['gpl'], 'pi30'],
+    DEV00000PI30Q: ['DE', ['gpl'], 'pi30q'],
```

Comment:
```suggestion
    DEV0000DEPI30: ['DE', ['gpl'], 'pi30'],
    DEV000DEPI30Q: ['DE', ['gpl'], 'pi30q'],
```
I'm thinking we change the dev accounts up and add 'DE' as a prefix before them because inevitably we will likely have Pi30 come to other countries as well and we'd want to differentiate our dev accounts by locale, similar to how we treat GPL.